tensor2tensor/data_generators/allen_downloader.py

```diff
@@ -0,0 +1,484 @@
+# coding=utf-8
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Data download utilities.
+
+Usage: python download.py --tmp_dir <temporary directory> ...
```

Comment:
`python -m tensor2tensor.data_generators.allen_downloader`

Also, why does this need to be run separately?

All of our datasets do their downloading as part of data generation with `t2t-datagen`.

Reply:
If you wanted to run N parallel data download jobs operating on different subsets of section list but that speed gain isn't really necessary - people can just run it overnight and Allen inst. may not want a large number of parallel requests anyway.